README.md

```diff
@@ -316,50 +323,6 @@ Finally, you can create an interactive session using the AWS CLI.  You'll need t
 aws ecs execute-command --cluster <your cluster ID> --container CaptureContainer --task <your task id> --interactive --command "/bin/bash"
 ```
 
-## How to run the unit tests & lint
-
-We now require that the ruff linter and unit tests pass before merging PRs.
-
-### Step 1 - Activate your Python virtual environment
-
-To isolate the Python environment for the project from your local machine, create virtual environment like so:
-```
-python3 -m venv .venv
-source .venv/bin/activate
-(cd manage_arkime ; pip install -r requirements.txt)
-```
-
-You can exit the Python virtual environment and remove its resources like so:
-```
-deactivate
-rm -rf .venv
-```
-
-Learn more about venv [here](https://docs.python.org/3/library/venv.html).
-
-### Step 2 - Run Pytest
-The unit tests are executed by invoking Pytest:
-
-```
-python -m pytest test_manage_arkime/
-```
-
-You can read more about running unit tests with Pytest [here](https://docs.pytest.org/en/7.2.x/how-to/usage.html).
-
-### Step 3 - Run Ruff
-The Python linter is executed by invoking Ruff:
-```
-ruff .
-```
-
-You can read more about Python linting with Ruff [here](https://beta.ruff.rs/docs/).
-
-### Step 4 - Run eslint
-The Typescript linter is executed by invoking [eslint]((https://eslint.org/):
-```
-npx eslint .
-```
-
 ## Performing CDK Bootstrap
```

Comment:
Should this go at the beginning near the "How to run the AWS All-in-One CLI" section?